spring-cloud-deployer-resource-maven/src/main/java/org/springframework/cloud/deployer/resource/maven/MavenProperties.java

```diff
@@ -220,6 +239,14 @@ public void setAuth(final Authentication auth) {
 
 		private String password;
 
+		public Authentication() {
+		}
+
+		public Authentication(String username, String password) {
```

Comment:
Same reason, although I should [apply it](https://github.com/donovanmuller/spring-cloud-deployer-openshift/blob/master/src/test/java/org/springframework/cloud/deployer/spi/openshift/factories/BuildConfigFactoryTest.java#L91)